rules/windows/defense_evasion_mshta_beacon.toml

```diff
@@ -1,15 +1,15 @@
 [metadata]
 creation_date = "2020/09/02"
 maturity = "production"
-updated_date = "2021/03/03"
+updated_date = "2021/07/19"
 
 [rule]
 author = ["Elastic"]
 description = """
 Identifies Mshta.exe making outbound network connections. This may indicate adversarial activity as Mshta is often
 leveraged by adversaries to execute malicious scripts and evade detection.
 """
-from = "now-9m"
+from = "now-120m"
```

Comment:
we should adjust the interval as well.
what's our default interval?

i think a healthy metric is for the interval to be >= 1/2 maxspan

Reply:
since these are process+network, i would say we just decrease maxspan to 10m

Reply:
default is `5m`. 

Nice suggestion, updating

Reply:
319e6db

test added for intervals in #1361

Reply:
```
E           AssertionError: The following rules have longer max_spans than lookbacks:
E           c2d90150-0133-451c-a783-533e736c12d7 - Mshta Making Network Connections -> lookback: 540000, maxspan: 7200000
E           f036953a-4615-4707-a1ca-dc53bf69dcd5 - Unusual Child Processes of RunDLL32 -> lookback: 540000, maxspan: 3600000
E           0d8ad79f-9025-45d8-80c1-4f0cd3c5e8e5 - Execution of File Written or Modified by Microsoft Office -> lookback: 540000, maxspan: 7200000
E           1defdd62-cd8d-426e-a246-81a37751bb2b - Execution of File Written or Modified by PDF Reader -> lookback: 540000, maxspan: 7200000
```